core/build.gradle.kts

```diff
@@ -116,3 +132,27 @@ publishing {
   apply<SigningPlugin>()
   configure<SigningExtension> { sign(publishing.publications) }
 }
+
+// TODO: when we enable parallelization for all sub-projects, we can extract the following block.
+tasks.test {
```

Comment:
Isn't this already set above?

Reply:
Duplicated code. Cleaned up and removed duplicated one.